src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/index.js

```diff
@@ -45,7 +45,11 @@ class Breakpoints extends Component {
     const { breakpointSources } = this.props;
 
     if (!breakpointSources.length) {
-      return null;
+      return (
+        <div className="no-breakpoints">
```

Comment:
Small nit, it's better to have classnames that describe the element. 

`no-breakpoints` mostly describes the state. For this, something like `empty-state-content` is more descriptive.

Reply:
all done